common/device-model.cpp

```diff
@@ -3022,8 +3028,7 @@ namespace rs2
                     const ImVec2 pos = ImGui::GetCursorPos();
 
                     draw_later.push_back([windows_width, &window, sub, pos, &viewer, this]() {
-                        if (!sub->streaming) ImGui::SetCursorPos({ windows_width - 34 , pos.y - 3 });
-                        else ImGui::SetCursorPos({ windows_width - 34, pos.y - 3 });
+                        ImGui::SetCursorPos({ windows_width - 41, pos.y - 3 });
```

Comment:
A `if/else` statement was removed because there `if` scope was identical to `else` scope.

Reply:
OK